infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationConfigManager.java

```diff
@@ -99,17 +91,38 @@ public LogReplicationConfigManager(CorfuRuntime runtime, String pluginConfigFile
         this.corfuStore = new CorfuStore(runtime);
 
         initStreamNameFetcherPlugin();
-        openTables();
+        openVersionTable();
+        unregisterStreamsNameTable(runtime);
```

Comment:
Let us add a comment explaining the purpose of this method.  Something like - 
"This method deletes and unregisters(removes from TableRegistry) the Streams Name Table, if it is present.
In prior versions, this table was used to store the names of streams to be replicated.  We no longer use this table so it can be completely removed if it exists".

Also, I suggest renaming the method to deleteAndUnregisterStreamsNameTable()